client/components/account-balances/balance-tooltip.tsx

```diff
@@ -18,10 +19,20 @@ const BalanceTooltip: React.FC< BalanceTooltipProps > = ( {
 	label,
 	content,
 } ) => {
+	const onClick = () => {
+		wcpayTracks.recordEvent(
+			wcpayTracks.events.OVERVIEW_BALANCE_TOOLTIP_CLICK,
```

Comment:
I don't know if we need to add events for tooltip clicks - we've been focusing on the core views and interactions, and didn't add view or click events for the various notices.

Reply:
To clarify, I suggest removing this event (and supporting code) unless it's important/required by product team to track interactions with the tooltip(s).

Reply:
Even better. Making this PR much simpler. Addressed here [ca88de8](https://github.com/Automattic/woocommerce-payments/pull/6307/commits/ca88de804a1fd82c607b62f6b66044daccb02aa4).